spring-cloud-gcp-starters/spring-cloud-gcp-starter-sql/src/main/java/org/springframework/cloud/gcp/sql/GcpCloudSqlProperties.java

```diff
@@ -68,6 +70,14 @@ public void setRegion(String region) {
 		this.region = region;
 	}
 
+	public String getInstanceConnectionName() {
+		return this.instanceConnectionName;
+	}
+
+	public void setInstanceConnectionName(String instanceConnectionName) {
+		this.instanceConnectionName = instanceConnectionName;
+	}
+
```

Comment:
What do you think of adding an optional `projectId` property here in case the database you're connecting to is in a different project?